docker-compose.yml

```diff
@@ -4,14 +4,20 @@ networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:2.0.1
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - ENGINE=mysql.connector.django
+      - NAME=app_db
+      - USER=app_user
+      - PASSWORD=1234
```

Comment:
The `PASSWORD` environment variable is set to '1234'. It's important to use a more secure password for production environments to ensure the security of your application.

Reply:
This is intended according to video lectures.